bower_components/aura/lib/ext/mediator.js

```diff
@@ -92,7 +91,6 @@ define('aura/ext/mediator', function () {
        * @param {Object} payload    Payload emitted
        */
       app.sandbox.emit = function () {
-        if (this.stopped) { return; }
```

Comment:
hmm i´m not sure if we should remove that ... this would enable emitting events on "lost" components ... i would take a look why you try to emit an event on a stopped component ...